kolibri/core/assets/test/views/auth-message.spec.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env mocha */
```

Comment:
Side note:

I freaking love that you write tests. I'm want to start doing it myself and I'm pumped to get some discussions going with the team on making this a team-wide standard.